### DISCUSSION TITLE
Modifications in 'Precursor Ion Generation'

### DISCUSSION THREAD

**Discussion by jiyoungyoun** [vdemichev/DiaNN#1259] [Q&A]
Hi, 

Under 'Precursor Ion Generation' section in the GUI, N-term excision and Carbamidomethylation is included as a default. Are these fixed mods or variable mods? It is not clear, since this is mixed with PTM mods that are used as variable mods (Acetylation, Phospho, di-GG).

Thank you!
Ji-Young

**Comment by vdemichev** [comment]
Hi Ji-Young,

Met excision is variable, C(cam) is fixed.

Best,
Vadim
